client/src/AppHub.js

```diff
@@ -32,34 +33,54 @@ const AppHub = () => (
                     <AlertsProvider>
                         <CssReset />
                         <CssVariables colors spacers />
-                        <Header />
-                        <main className={styles.main}>
-                            <Switch>
-                                <Route exact path="/" component={Apps} />
-                                <Route path="/app/:appId" component={AppView} />
-                                <Route
-                                    path="/organisation/:organisationSlug/view"
-                                    component={OrganisationView}
-                                />
-                                <ProtectedRoute
-                                    path="/user"
-                                    auth={Auth}
-                                    component={UserView}
-                                />
-                                <ProtectedRoute
-                                    path="/verify/org/callback"
-                                    auth={Auth}
-                                    component={OrganisationInvitationCallback}
-                                />
-                                <Route
-                                    exact
-                                    path="/verify/org"
-                                    component={OrganisationInvitation}
-                                />
-                                {/* No-match route - redirect to index */}
-                                <Route render={() => <Redirect to="/" />} />
-                            </Switch>
-                        </main>
+                        <Switch>
+                            <Route
```

Comment:
this is a route that doesn't have the shell (the headre and sidebar..) of AppHub, that can be integrated as an iframe in app-management